plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/editor/context/project/ProjectContextProvider.kt

```diff
@@ -261,23 +264,35 @@ class ProjectContextProvider(val project: Project, private val encoderServer: En
     }
 
     private fun isBuildOrBin(filePath: String): Boolean {
-        val regex = Regex("""[/\\](bin|build|node_modules|venv|env|\.idea)[/\\]""", RegexOption.IGNORE_CASE)
+        val regex = Regex("""[/\\](bin|build|node_modules|venv|.venv|env|\.idea)[/\\]""", RegexOption.IGNORE_CASE)
```

Comment:
why don't we also ask the IDE what SDK it's using and just exclude it

Reply:
do you mean to exclude the associated common build dirs for the specific SDK?

does this info come from JB APIs or we have to build it on our own?

Reply:
JB APIs can return SDK root,
APIs also have build directories, but another option would be to query if the directory has been excluded